resources/definition.rb

```diff
@@ -30,21 +48,40 @@
     package package_deps
   end
 
-  definition_path = ::File.join(new_resource.prefix_path, new_resource.definition)
+  installation_path = if new_resource.version_prefix
+                        ::File.join(new_resource.prefix_path, new_resource.definition)
+                      else
+                        new_resource.prefix_path
+                      end
 
-  if platform_family?('mac_os_x')
-    env = { 'RUBY_CONFIGURE_OPTS' => "--with-openssl-dir=#{openssl_prefix.to_s}" }
-    env.merge(new_resource.environment) unless new_resource.environment.nil?
```

Comment:
What I was trying to do here was, allow the user not to set the environment variables, and always have `RUBY_CONFIGURE_OPTS` set. This way ruby-install detects there is a version of openssl installed, and therefore doesn't try to install another copy. 

I suppose another way would be to just set a default environment property on `mac_os_x`?

Reply:
Sure, and this change honors that. IMHO it's just slightly better to have the `if` return the value of `env` rather than do assignments within the `if` statement. It's not that big of a deal to me, really just a style thing. (And style is always debatable.)